assets/src/edit-story/components/canvas/useCanvasSelectionCopyPaste.js

```diff
@@ -146,14 +153,26 @@ function useCanvasSelectionCopyPaste(container) {
         for (let i = 0; i < items.length; i++) {
           const item = items[i];
           if (isValidType(item)) {
-            uploadFile(item.getAsFile());
+            try {
+              uploadFile(item.getAsFile());
+            } catch (e) {
+              if (!e.isUserError) {
+                e.message = __(
+                  'Sorry, files has failed to upload',
```

Comment:
```suggestion
                  'Sorry, files have failed to upload',
```

Reply:
Fixed. It was supposed to be `file has failed...` 😄

Reply:
Not fixed yet?

Reply:
Fixed!